Dockerfile

```diff
@@ -48,6 +48,7 @@ RUN curl -L https://github.com/jmespath/jp/releases/download/${JP_VERSION}/jp-li
 WORKDIR azure-cli
 COPY . /azure-cli
 
+RUN pip install ./privates/azure_mgmt_containerregistry-3.0.0rc8-py2.py3-none-any.whl
```

Comment:
well. Looks like it's just a private package.

Reply:
@adewaleo, make sure revert it before merge. We must publish the SDK package beforehand